staging/src/k8s.io/pod-security-admission/policy/registry.go

```diff
@@ -78,18 +79,28 @@ func (r *checkRegistry) EvaluatePod(lv api.LevelVersion, podMetadata *metav1.Obj
 }
 
 func validateChecks(checks []Check) error {
-	ids := map[string]bool{}
+	ids := map[string]sets.String{}
 	for _, check := range checks {
-		if ids[check.ID] {
-			return fmt.Errorf("multiple checks registered for ID %s", check.ID)
-		}
-		ids[check.ID] = true
 		if check.Level != api.LevelBaseline && check.Level != api.LevelRestricted {
 			return fmt.Errorf("check %s: invalid level %s", check.ID, check.Level)
 		}
 		if len(check.Versions) == 0 {
 			return fmt.Errorf("check %s: empty", check.ID)
 		}
+
+		levels, ok := ids[check.ID]
+		if ok {
+			if levels.Has(string(check.Level)) {
```

Comment:
We didn't want to use the same ID for distinct baseline and restricted checks. @tallclair, what would you suggest for ids for these two checks? "baselineSeccomp" and "restrictedSeccomp" are descriptive, at least

Reply:
should be able to revert the changes to this file now